File: RealEstate-Source-Code-v6.0/AdminPanel-NetCore/Helpers/FileHelper.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;

namespace RealEstate.Helpers
{
    public static class FileHelper
    {
        public static bool checkCacheFile(string imageName)
        {
            var path = Path.Combine(Directory.GetCurrentDirectory(), "UploadFiles/Caches", imageName);
            return File.Exists(path);
        }

        public static bool checkImageFile(string imageName)
        {
            var path = Path.Combine(Directory.GetCurrentDirectory(), "UploadFiles/Images", imageName);
            return File.Exists(path);
        }
        public static void moveCacheToImages(string imageName)
        {
            var source = Path.Combine(Directory.GetCurrentDirectory(), "UploadFiles/Caches", imageName);
            var dest = Path.Combine(Directory.GetCurrentDirectory(), "UploadFiles/Images", imageName);
            if (File.Exists(source))
            {
                File.Move(source, dest);
            }
        }

        public static void removeImageFile(string imageName)
        {
            var path = Path.Combine(Directory.GetCurrentDirectory(), "UploadFiles/Images", imageName);
            if (File.Exists(path))
            {
                File.Delete(path);
            }
        }
    }
}

File: RealEstate-Source-Code-v6.0/AdminPanel-Laravel10/app/Helpers/FileHelper.php
<?php

namespace App\Library;

use Illuminate\Support\Facades\File;

class FileHelper
{
    public static function checkImageFile(string $imageName)
    {
        $imagePath = public_path('/uploadfiles/images/') . $imageName;
        return File::exists($imagePath);
    }

    public static function moveCacheToImages(string $imageName)
    {
        $sourceImagePath = public_path('/uploadfiles/caches/') . $imageName;
        $destImagePath = public_path('/uploadfiles/images/') . $imageName;
        if (File::exists($sourceImagePath)) {
            File::move($sourceImagePath, $destImagePath);
        }
    }

    public static function removeImageFile(string $imageName)
    {
        $imagePathForCaches = public_path('/uploadfiles/caches/') . $imageName;
        $imagePathForImages = public_path('/uploadfiles/images/') . $imageName;
        if (File::exists($imagePathForCaches)) {
            File::delete($imagePathForCaches);
        }
        if (File::exists($imagePathForImages)) {
            File::delete($imagePathForImages);
        }
    }
}